src/transformStateWithClones.js

```diff
@@ -8,6 +8,33 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const currentState = { ...state };
+  const stateChangesArr = [];
```

Comment:
Don't use data type names in variable names
```suggestion
  const stateChanges = [];
```